cont3xt/overview.js

```diff
@@ -76,14 +76,89 @@ class Overview {
         Db.putOverview(iType, createDefaultForIType(iType)).then(() => {
 
         }).catch((err) => {
-          console.log(`ERROR - failed to create default overview for ${iType} iType... skipping`, err);
+          console.log(`WARNING - failed to create default overview for ${iType} iType... skipping`, err);
         });
       }).catch((err) => {
-        console.log(`ERROR - failed to check existence of ${iType} iType default overview... skipping:`, err);
+        console.log(`WARNING - failed to check existence of ${iType} iType default overview... skipping:`, err);
       });
     }
   }
 
+  // Verify the given overview, returns error { msg: string } on failure, { custom } otherwise
+  static verifyOverviewCustomField (custom) {
```

Comment:
this and similar verifiers seem pretty boiler-platey to me. Out of curiousity, have we ever considered schema-based validation (like zod, yup, etc)? We're pretty much writing our own z.object.safeParse (returns a clean validated object or an error object) from zod here, so I wonder if that might be preferred?—though I suppose that would mean a new outside dependency, which we maybe want to avoid?

Reply:
I have not considered this. I agree that this could be "outsourced" to another tool, however I'd have to investigate whether this is worth the time/effort. I think this is something to add to our list of things to improve upon, but not something we want to do right now.

Reply:
Yea, I just thought it might be a good consideration for the future—seeing as we've been doing more of this sort of validation lately. I personally like the more declarative style of zod, especially the way you can mark fields optional versus required and handle properties that might be different types—plus we wouldn't need to use the funny immediately-invoked lambda anymore 😆.

Reply:
Agreed. I'll put it on my list of things to look in to! I think it could be beneficial.

Reply:
I had looked at some before but it seemed like you were just programming them in another DSL, but probably would be a good idea and less mistakes. :)